test/keyboard-navigation.html

```diff
@@ -187,17 +187,30 @@
             expect(overlay.focusedDate).to.eql(new Date(1999, 11, 25));
           });
 
-          it('should focus one day forward with arrow right', () => {
-            arrowRight();
-
-            expect(overlay.focusedDate).to.eql(new Date(2000, 0, 2));
-
-          });
-
-          it('should focus one day backward with arrow left', () => {
-            arrowLeft();
-
-            expect(overlay.focusedDate).to.eql(new Date(1999, 11, 31));
+          ['ltr', 'rtl'].forEach(direction => {
+            describe(`[dir="${direction}"]`, () => {
+              const isRTL = direction === 'rtl';
+              before(() => {
+                document.documentElement.setAttribute('dir', direction);
+              });
+
+              after(() => {
+                if (isRTL) {
+                  document.documentElement.setAttribute('dir', 'ltr');
```

Comment:
Nit: can just remove the attribute. Not a blocker.

Reply:
I'd rather force it to `ltr` because of the Safari issue I faced in context-menu.